app/services/selection/selection.ts

```diff
@@ -95,7 +97,19 @@ export class SelectionService
 
   @shortcut('Delete')
   remove() {
-    return this.getSelection().remove.call(this);
+    const name = this.getLastSelected().name;
+    electron.remote.dialog.showMessageBox(
+      electron.remote.getCurrentWindow(),
+      {
+        type: 'warning',
+        message: $t('Are you sure you want to remove %{sceneName}?', { sceneName: name }),
```

Comment:
I kept this as `sceneName` so we wouldn't need a new translation key for what amounts to non-visible text, even though it isn't 100% descriptive anymore. Changing this and the one in `SceneSelector` to be `name` would be an alternative.

Reply:
Ah, good thinking.